src/wrapper.cc

```diff
@@ -124,16 +124,16 @@ Dart_CObject CallbackArgBuilder(int num, ...) {
   Dart_CObject c_request_data;
   va_list valist;
   va_start(valist, num);
-  void *request_buffer = malloc(sizeof(uint64_t) * num);
-  uint64_t *buf = reinterpret_cast<uint64_t *>(request_buffer);
+  void *request_buffer = malloc(sizeof(intptr_t) * num);
+  intptr_t *buf = reinterpret_cast<intptr_t *>(request_buffer);

```

Comment:
This should be kelt uint64_t if you're using it only with asUint64List in Dart.

We can swap everything to be word-size (4 byte words or 8 byte words) specific when we start supporting 32 bit.

Reply:
Okay :)